src/jvmMain/kotlin/io/beatmaps/util/scheduled.kt

```diff
@@ -30,7 +30,12 @@ class CheckScheduled(private val rb: RabbitMQInstance) : TimerTask() {
                 ).mapNotNull {
                     schedulerLogger.info { "Scheduler publishing ${it.hash}" }
                     runBlocking { delay(1L) }
-                    if (publishVersion(it.mapId.value, it.hash, rb)) it else null
+                    // Undeceiver: I am not adding the "alert" parameter to the database because it feels like too much effort for
+                    // the unlikely case that someone scheduled a map for release but doesn't want to notify followers,
+                    // If this ever happens, the alert will go out regardless (if it's scheduled it's likely it's a big deal).
+                    // If we did want to consider the parameter, it can be passed here without any problem,
+                    // but then we'd have to save it in the database when scheduling. Can be done, not doing it now.
```

Comment:
Should probably not leave this comment in, but it is useful as a comment on the PR

An alternative (maybe easier, idk) approach could be to add a new state of `ScheduledWithAlert` or similar that triggers the alert but the "normal" `Scheduled` doesn't

Reply:
Copy/pasting the comment here then.

I am not adding the "alert" parameter to the database because it feels like too much effort for the unlikely case that someone scheduled a map for release but doesn't want to notify followers. If this ever happens, the alert will go out regardless (if it's scheduled it's likely it's a big deal). If we did want to consider the parameter, it can be passed here without any problem, but then we'd have to save it in the database when scheduling. Can be done, not doing it now.

Reply:
The reason I added it to the code was in case somebody stumbles upon this in the future, looks at that `true` and wonders why the hell is the alert flag not passed on. But I figure they can probably trace it back to the commit/PR if it happens.